docs/table-upgrade.md

```diff
@@ -0,0 +1,41 @@
+# Table Upgrade logic and data structures
+The Hive Metastore migration process will upgrade the following Assets:
+- Tables ond DBFS root
+- External Tables
+- Views
+
+We don't expect this process to be a "one and done" process. This typically is an iterative process and may require a few runs.
+
+We suggest to keep track of the migration and provide the user a continuous feedback of the progress and status of the upgrade.
```

Comment:
please elaborate how this continuous feedback should be delivered to the customer, if we're migrating 300 or 3000 workspaces.